File: angular/src/app/shared/service/product/models/product-dto.ts
import { EntityDto } from '@abp/ng.core';

export class ProductDto extends EntityDto<string>{
  productName: string;
  productDescription: string;
  CategoryId: string;
  price: number;
  quantity: number;
  imgSource: string;

  constructor(initial: Partial<ProductDto> = {}) {
    super(initial);
  }
}

File: aspnet-core/src/Selling_Website.Application.Contracts/Products/Dto/ProductDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace Selling_Website.Products.Dto
{
    public class ProductDto : FullAuditedEntityDto<Guid>
    {
        public string ProductName { get; set; }
        public string ProductDescription { get; set; }
        public Guid? CategoryId { get; set; }
        public int? Price { get; set; }
        public string Quantity { get; set; }
        public string ImgSource { get; set; }
}
}
